### ISSUE TITLE
Errors with Hysteresis and Canny 3D segmentation

### ISSUE THREAD

**Issue by hansenjn** [hansenjn/CiliaQ_Preparator#4]
Recently errors occur when applying "HYSTERESIS threshold" or "CANNY3D" as the "Segmentation method", but only when the image was loaded through the multi-task manager processing option ( <img src="https://github.com/hansenjn/CiliaQ_Preparator/assets/27991883/6d6b70dc-0f6c-4f09-9b56-03ea071c19e3" height=24> ).

For CANNY 3D, two error messages appear:
Message 1:
```
java.lang.NullPointerException
	at ciliaQ_Prep_jnh.canny3d_thresholder.Processing.doProcessing(Processing.java:52)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.segmentUsingCanny3D(CiliaQPrepMain.java:1437)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.run(CiliaQPrepMain.java:567)
	at ij.IJ.runUserPlugIn(IJ.java:244)
	at ij.IJ.runPlugIn(IJ.java:210)
	at ij.Executer.runCommand(Executer.java:152)
	at ij.Executer.run(Executer.java:70)
	at java.lang.Thread.run(Thread.java:750)
```

Message 2:
```
java.lang.NullPointerException
	at mcib_plugins.Hysteresis_Thresholding.run(Hysteresis_Thresholding.java:43)
	at ij.plugin.filter.PlugInFilterRunner.processOneImage(PlugInFilterRunner.java:266)
	at ij.plugin.filter.PlugInFilterRunner.<init>(PlugInFilterRunner.java:114)
	at ij.IJ.runUserPlugIn(IJ.java:246)
	at ij.IJ.runPlugIn(IJ.java:210)
	at ij.Executer.runCommand(Executer.java:152)
	at ij.Executer.run(Executer.java:70)
	at ij.IJ.run(IJ.java:326)
	at ij.IJ.run(IJ.java:416)
	at ciliaQ_Prep_jnh.canny3d_thresholder.Processing.doProcessing(Processing.java:50)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.segmentUsingCanny3D(CiliaQPrepMain.java:1437)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.run(CiliaQPrepMain.java:567)
	at ij.IJ.runUserPlugIn(IJ.java:244)
	at ij.IJ.runPlugIn(IJ.java:210)
	at ij.Executer.runCommand(Executer.java:152)
	at ij.Executer.run(Executer.java:70)
	at java.lang.Thread.run(Thread.java:750)
```

For applying the HYSTERESIS threshold, one error messages appears:

```
java.lang.NullPointerException
	at ciliaQ_Prep_jnh.CiliaQPrepMain.doHysteresisThreshold(CiliaQPrepMain.java:1523)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.segmentUsingHysteresis(CiliaQPrepMain.java:1486)
	at ciliaQ_Prep_jnh.CiliaQPrepMain.run(CiliaQPrepMain.java:580)
	at ij.IJ.runUserPlugIn(IJ.java:217)
	at ij.IJ.runPlugIn(IJ.java:181)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:750)
```

The errors must relate to changes in the 3DSuite library that CiliaQ Preparator uses for performing operations in the segmentation workflows for "HYSTERESIS threshold" or "CANNY3D" since these errors only emerge recently.

**Comment by hansenjn** [comment]
With newer versions of the 3DSuite it seems to be a requirement that the image is displayed before running the 3D Hysteresis threshold method from the 3DSuite. In the next version (v0.1.2) this will be implemented and thereby, the issue will be fixed.

**Comment by hansenjn** [comment]
Fixed with the release of version v0.1.2: https://github.com/hansenjn/CiliaQ_Preparator/releases
